Cargo.toml

```diff
@@ -56,6 +56,8 @@ json = ["serde_json"]
 
 multipart = ["mime_guess"]
 
+tower = ["tower-service"]
```

Comment:
I don't think we need to make this optional, hyper already depends on it.

Reply:
Sure!

Reply:
Done